front/temporal/labs/workflows.ts

```diff
@@ -48,8 +48,5 @@ export async function processTranscriptWorkflow({
   fileId: string;
   transcriptsConfigurationId: ModelId;
 }): Promise<void> {
-  await processGoogleDriveTranscriptActivity(
-    transcriptsConfigurationId,
-    fileId
-  );
+  await processTranscriptActivity(transcriptsConfigurationId, fileId);
```

Comment:
Does it require a restart of the workflows?

Reply:
I wouldn't think so - seems like the cron workflows are not long running if I'm not mistaken but creating a new one at each run

Reply:
awesome!